tck/src/main/scala/io/cloudstate/tck/CloudStateTCK.scala

```diff
@@ -169,52 +169,54 @@ class CloudStateTCK(description: String, settings: CloudStateTCK.Settings)
         spec.entities.find(_.serviceName == CrdtTckModel.name).foreach { entity =>
           serviceNames must contain("CrdtTckModel")
           entity.entityType mustBe Crdt.name
+          entity.passivationStrategy must not be None
         }
 
         spec.entities.find(_.serviceName == CrdtTwo.name).foreach { entity =>
           serviceNames must contain("CrdtTwo")
           entity.entityType mustBe Crdt.name
+          entity.passivationStrategy must not be None
         }
 
         spec.entities.find(_.serviceName == EventSourcedTckModel.name).foreach { entity =>
           serviceNames must contain("EventSourcedTckModel")
           entity.entityType mustBe EventSourced.name
           entity.persistenceId mustBe "event-sourced-tck-model"
-          entity.passivationStrategy must (not be None or be(None))
+          entity.passivationStrategy must not be None
         }
 
         spec.entities.find(_.serviceName == EventSourcedTwo.name).foreach { entity =>
           serviceNames must contain("EventSourcedTwo")
           entity.entityType mustBe EventSourced.name
-          entity.passivationStrategy must (not be None or be(None))
+          entity.passivationStrategy must not be None
         }
 
         spec.entities.find(_.serviceName == EventSourcedShoppingCart.name).foreach { entity =>
           serviceNames must contain("ShoppingCart")
           entity.entityType mustBe EventSourced.name
           entity.persistenceId must not be empty
-          entity.passivationStrategy must (not be None or be(None))
+        //entity.passivationStrategy must not be None // some language support does not support passivation yet!!
```

Comment:
This would then show this test "green" for language supports not implementing passivation Strategy? 

If yes, perhaps this should show a failed test. The TCK should show which language support libraries are up-to-date. The TCK will probably show for some time a lot of "red and orange" for other implementations
https://observablehq.com/@marcellanz/cloudstate-tck-verification as the protocol progresses.

Reply:
Good point. I will rather try to put the test on pending for language support not implementing the passivation strategy